Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -377,6 +378,17 @@ void WindowManager::tell_wms_applet_area_size_changed(const Gfx::IntSize& size)
     });
 }
 
+void WindowManager::tell_wms_super_key_pressed()
+{
+    for_each_window_manager([&](WMClientConnection& conn) {
```

Comment:
do we actually need to capture anything? this only uses `conn`, which is passed as an argument.

Reply:
Oops... I'll fix that now 😅